### PR TITLE
KNX Multicast without KNXD

### PR DESCRIPTION
**Pull request by c-o-m-m-a-n-d-e-r**
Hey,
i think with these changes it will be possible to use a multicast route without KNXD.

Please take a look. Don't sure if i miss something

### CODE REVIEW

**Comment by snowdd1** [comment]
Sorry to be so late, I will try this, and if it works both ways, I will merge it!

**Comment by c-o-m-m-a-n-d-e-r** [comment]
no problem ... i'm not 100% sure but i think there are some bugs i found a little bit later ...
Please check.

File: index.js [hunk 1]
```diff
@@ -288,11 +290,7 @@ KNXPlatform.prototype.configure = function() {
 			if (that.config.AllowKillHomebridge===true) {
 				response.write(' <br><hr><br><a href="/kill">Kill homebridge</a> by throwing an Error. Use this to restart HomeBridge if you have it configured as a self-starting service ' + ' <BR>');
 			}
-			response.write(`<HR><BR>Available pages are <br>
-					<a href="/list">list devices</a> and <br>
-					<a href="/availservices">list available services</a><br>
-					 <a href="/availcharacteristics">list available characteristics</a>
-					`);
+			response.write('<HR><BR>Available pages are <br><a href="/list">list devices</a> and <br><a href="/availservices">list available services</a><br><a href="/availcharacteristics">list available characteristics</a>');
```
Comment (snowdd1) [inline index.js]: Why did you convert all multi-line strings to single line strings? Some issue with your IDE?

Comment (c-o-m-m-a-n-d-e-r) [reply index.js]: Ops, i think there was a plugin in my IDE over motivated ...

**Pull request merged**